portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,60 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+// Function that shows or hides the list of courses
+// for a particular semester upon user click
+function toggleCourseList(listObject) {
+    let semester = listObject.id;
+    let semCourseList = "courselist" + semester;
+    let lst = document.getElementById(semCourseList);
+    if(lst.style.display === "none" || lst.style.display === "")
+    {
+        lst.style.display = "block";
+    }
+    else
+    {
+        lst.style.display = "none";
+    }
+}
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// function that changes slideshow's picture to the next 
+// picture when the user presses the next button
+function next() {
```

Comment:
nit: The next() and prev() functions have a few shared functionalities, which can be made into helper functions. i.e. I could imagine a "getImageNum" function that gets the imgNum.